docs/node/run-your-node/paratime-client-node.mdx

```diff
@@ -216,6 +216,38 @@ accessible on the internet (for `TCP` and `UDP` traffic).
 
 :::
 
+## Configuring TEE Paratime Client Node
+
+If your node requires the ability to issue queries that can access confidential
+data, start by following the [Configuration](#configuration) section to create
+the `/node/etc/config.yml` file. Once the file is set up, add the following
+content to the `runtime` part within the `/node/etc/config.yml` file:
+
+**Update Runtime Configuration:**
+
+   ```yaml
+   runtime:
+       # Paths to ParaTime bundles for all of the supported ParaTimes.
+       paths:
+           - {{ runtime_orc_path }}
+       environment: sgx
+       config:
+         {{ paratime_id }}:
+             estimate_gas_by_simulating_contracts: true
+             allowed_queries:
+                 - all_expensive: true 
```

Comment:
This is unrelated to running "TEE client nodes", rather it is required in case you want to issue certain runtime queries (e.g. for running the Web3 Gateway). I'm not sure if this should be here, as we already mention this in the [Web3 Gateway chapter](https://docs.oasis.io/node/web3#oasis-paratime-client-node)? We could leave it in this chapter, but move it outside the "TEE client nodes" section, rather just have a separate section about "Enabling expensive queries".